nims-notes-things/src/components/notes.js

```diff
@@ -11,12 +11,17 @@ function Notes() {
 
   const [sorted, setSorted] = useState(false);
```

Comment:
No need for this state anymore if it is being handled with Redux

Reply:
Ahh - see now it makes sense why it was working lol i hadn't removed the line